.github/workflows/test.yml

```
@@ -85,3 +85,21 @@ jobs:
       - name: Run unit tests with tox (macOS)
         if: startsWith(matrix.platform, 'macos')
         run: tox -e py-unit
+
+  man:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

All workflows now check out the full history + submodules.
```suggestion
      - uses: actions/checkout@v4
        with:
          fetch-depth: 0
          submodules: true
```